### PR TITLE
Changed screenshots

### PR DESCRIPTION
**Pull request by saitama42069**
updated the screenshots.

### CODE REVIEW

**Comment by saitama42069** [comment]
> Fix the conflicts

I did. It was a typo in the readme.md and actual file name of the screenshots.

File: Onepunch/user.css [hunk 1]
```diff
@@ -201,13 +201,20 @@ html .SidebarList__title {
...
+.LeftSidebar__section {
+    margin-bottom: -30px;
+    margin-top: 55px;
+}
+.sidebar .sidebar-navbar {
+    background-image: url(https://local_resource_host/images/spotify-logo-primary-green.png);
+    background-repeat: no-repeat;
+    background-size: 210px;
+    background-position-x: 20px;
+}
```
Comment (saitama42069) [inline Onepunch/user.css:217]: I'll update this next time

**Comment by morpheusthewhite** [comment]
> > Fix the conflicts
> 
> I did. It was a typo in the readme.md and actual file name of the screenshots.

Yet I am not able to merge due to conflicts

File: Onepunch/user.css (hunk 1 above)
Comment (morpheusthewhite) [reply Onepunch/user.css:217]: What do you mean? You can still commit changes since I didn't merged yet

**Comment by saitama42069** [comment]
Now, it'll be fine I guess

**Pull request merged**